shared/common-adapters/swipeable.native.tsx

```diff
@@ -239,6 +248,15 @@ export const SwipeTrigger = React.memo(function SwipeTrigger(p: {
         resetPosition()
       },
       onPanResponderTerminate: () => {
```

Comment:
this can happen even though we've actually done a real swipe, so handle it here also